airbyte-webapp/src/packages/cloud/views/users/AccountSettingsView/components/PasswordSection/typings.d.ts

```diff
@@ -0,0 +1,5 @@
+export type FormValues = {
```

Comment:
No need to use d.ts extensions here as .d.ts is bridge between ts and js. We use just simple ts.